tests/equisolve_tests/utils/test_split_data.py

```diff
@@ -0,0 +1,834 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2023 Authors and contributors
+# (see the AUTHORS.rst file for the full list of names)
+#
+# Released under the BSD 3-Clause "New" or "Revised" License
+# SPDX-License-Identifier: BSD-3-Clause
+from typing import List
+
+import equistore
+import numpy as np
+import pytest
+from equistore import Labels, TensorBlock, TensorMap
+from numpy.testing import assert_equal
+
+from equisolve.utils import split_data
+
+
+@pytest.fixture
+def test_tensor_map_a():
+    """
+    Create a dummy tensor map to be used in tests. This is the same one as the
+    tensor map used in `tensor.rs` tests.
+    """
```

Comment:
We have to generate some test tensormaps. It is super hard to read the actual tests when they are blown up with these lines.

Reply:
To be discussed in the upcoming dev meeting